### PR TITLE
Add Vuepress full-text search ultimate version-algolia's open source implementation meilisearch full access guide

### PR DESCRIPTION
**Pull request by eryajf**
# Pull Request

## Related issue

https://github.com/meilisearch/meilisearch/discussions/3535

## What does this PR do?

add a Community Blog Posts

`Because the original text of this article is in Chinese, the name of the resource is also in Chinese, while the following comments are annotated in English. I hope it's not against the rules.`

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,7 @@ Please take a look at the [Contribution Guidelines](https://github.com/meilisear
 
 ### Community Blog Posts
 * [Meilisearch: A Minimalist Full-Text Search Engine](https://tech.marksblogg.com/meilisearch-full-text-search.html)
+* :cn: [Vuepress全文搜索终极版-algolia的开源实现meilisearch全接入指南](https://wiki.eryajf.net/pages/dfc792/) - Vuepress full-text search ultimate version-algolia's open source implementation meilisearch full access guide
```
Comment (ferdi05) [inline README.md:222]: Hi @eryajf thanks for your contribution.
I must say that I find it hard to understand the title/description. We don't think ourselves as an implementation of Algolia, but as an alternative. So I find it quite confusing.
Would you ke ok to change it somehow? Thanks

**Review by ferdi05 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eryajf) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ Please take a look at the [Contribution Guidelines](https://github.com/meilisear
 
 ### Community Blog Posts
 * [Meilisearch: A Minimalist Full-Text Search Engine](https://tech.marksblogg.com/meilisearch-full-text-search.html)
-* :cn: [Vuepress全文搜索终极版-algolia的开源实现meilisearch全接入指南](https://wiki.eryajf.net/pages/dfc792/) - Vuepress full-text search ultimate version-algolia's open source implementation meilisearch full access guide
+* :cn: [Vuepress全文搜索终极版-基于开源搜索引擎meilisearch的全接入指南](https://wiki.eryajf.net/pages/dfc792/) - Vuepress Full Text Search Ultimate Edition-Full Access Guide Based on Open Source Search Engine meilisearch
```
Comment (ferdi05) [inline README.md:222]: Thanks for the update. I think using `Full` this often is a bit overkill. 
 How about `A guide on Meilisearch-based full-text search in Vuepress`? I could be super wrong on your intent though, I must say I don't read Chinese and I could be wrong about what's in the guide

**Review by ferdi05 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -219,7 +219,7 @@ Please take a look at the [Contribution Guidelines](https://github.com/meilisear
 
 ### Community Blog Posts
 * [Meilisearch: A Minimalist Full-Text Search Engine](https://tech.marksblogg.com/meilisearch-full-text-search.html)
-* :cn: [Vuepress全文搜索终极版-基于开源搜索引擎meilisearch的全接入指南](https://wiki.eryajf.net/pages/dfc792/) - Vuepress Full Text Search Ultimate Edition-Full Access Guide Based on Open Source Search Engine meilisearch
+* :cn: [Vuepress全文搜索终极版-基于开源搜索引擎meilisearch的接入配置指南](https://wiki.eryajf.net/pages/dfc792/) - A guide on Meilisearch-based full-text search in Vuepress
```
Comment (ferdi05) [inline README.md:222]: Thanks for the changes. I'm sorry for being annoying, but I just thought that as a guide it should probably belong to the `Community guides` section? If you agree, I'll make the changes, you've already done a lot :)

**Pull request merged**